docs/release/trg-0/trg-1-07.md

```diff
@@ -0,0 +1,30 @@
+---
+title: TRG 1.07 - End-User Manual (User Assistance)
+sidebar_position: 1
+---
+
+:::caution
+Proposed release date: 15th of January 2024
+:::
+
+| Status     | Created      | Post-History                           |
+|------------|--------------|----------------------------------------|
+| Draft      | 31-Jan-2024  | created initial Draft                  |
+
+## Why
+
+End-User Manual is essential for providing user assistance and ensuring a positive user experience. The user of the application is the receipient of this documentation. Typical contents are screenshots, description of functionality and steps that guide through critical processes.
+
+## Description
```

Comment:
Is the End-User Manual relevant for all components? If not, I would describe that at the beginning of the "Description" section

Reply:
Users of the application are the recipients of this documentation. In case as application by default doesn`t address  end-users, an enduser manual is not required.
The Enduser manual is essential for providing user assistance and ensuring a positive user experience. Typical contents are screenshots, description of functionality and steps that guide through critical processes.